wiki/Getting_Started_Contributing_to_Wiki/en.md

```diff
@@ -0,0 +1,182 @@
+
+![GitHub Preview](7.JPG "Github Preview")
+
+Now Click on the Pencil Icon to edit. And it should redirect you to the Web Editor.
+
+![GitHub's Web Editor](8.JPG "web Editor Snapshot")
+
+If you want to preview your changes, click the "Preview Changes" Tab. To go back to editing, click the "Edit File" Tab
+
+After making your changes, scroll down and Fill the following fields to explain your edit(s) to other contributors.
+
+![Commit Message](9.JPG "Commit Message")
+
+_**Note: Your Commit Messages must explain thoroughly your changes, otherwise, it would not get merged**._
+
+You would be given a option where to save your edits.
+
+- **"Commit Directly to the ``xyz`` branch"** 
+  - will commit the changes through your branch directly.
+
+- **"Create a New Branch"** 
+  - automatically creates a Branch for your edits. This is useful if you're making bulk edits.
+
+Pick a option and click "Commit Changes".
+
+### Through your Local Machine
+
+GitHub under the hood is a Git Service. You can clone your fork in your Machine and make changes there.
+
+if you have git installed, put this in your Command Line Interface (CLI) :
+```prolog
+git clone https://github.com/<username>/osu-wiki 
+```
+and this should clone your fork into your specified folder.
+
+Now after making your changes locally, you will need to commit this on your local copy.
+
+this can be done by entering : 
+```prolog
+git add .
+git commit -m "<commit message here>"
+```
+However, in some editors (ie. Visual Studio Code), they provide a Git interface.
+
+![Visual Studio code's Git Interface](10.JPG "Visual Studio's Git Interface")
+
+After Committing your changes, we need to sync this on GitHub, to be able to do this, Enter the following in your Command Line Interface (CLI) :
+
+```prolog
+git push origin <branch>
+```
+It would prompt you to enter your login details, then it should push to GitHub.
+
+_Note: However, If you're authentication details were saved in Git already, no prompt will appear and will proceed directly to push._
+
+## Pull Requests
+
+After making your changes, you must make a "Pull Request" to be able for your changes to be made into the live osu!wiki.
+
+Go to the "Pull Requests" tab.
+
+![Pull Request Tab](6.JPG "Pull Request Tab")
+
+This would lead you to the Pull Requests page.
+
+![PR UI](11.jpg "Pull Requests Page")
+
+This is the Pull Request Page. Now click on "New Pull Request"
+
+![Compare Changes Page](12.jpg "Comparing Changes")
+
+Now we want to compare ``ppy:master`` against your branch.
+
+Now select where did you made your changes in "branch" beside your "head Fork".
+
+![Comparing Changes 2](13.jpg "Comparing Changes 2")
+
+_However if you made changes in your Master Branch, The Changes will be shown without changing anything in the head fork options_.
+
+Now after viewing your changes, click "Create Pull Request" and fill it with the following details:
+
+- **Title** : The Summary of the Changes you've made.
```

Comment:
change to
```
- **Title** : the summary of the changes you have made
  - Note: it is really important to have a meaningful and clear title, otherwise it will cause confusion
```